tests/test-raxmon-assign-agent-to-entity.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Where is this playbook used?

Reply:
This one is not used in the regular playbook (site.yml),  really just here for testing just in case, we need to create an entity.  
Same for the delete entity play.